src/components/frame/Extensions/pages/Extensions/ExtensionsListingPage/ExtensionsListingPage.tsx

```diff
@@ -19,34 +24,44 @@ const MSG = defineMessages({
     id: `${displayName}.availableExtensions`,
     defaultMessage: 'Available Extensions',
   },
+  decisionMethods: {
+    id: `${displayName}.decisionMethods`,
+    defaultMessage: 'Decision methods',
+  },
+  payments: {
+    id: `${displayName}.payments`,
+    defaultMessage: 'Payments',
+  },
 });
 
 const ExtensionsListingPage: FC = () => {
   const { availableExtensionsData, installedExtensionsData } =
     useExtensionsData();
-
   useSetPageHeadingTitle(formatText(MSG.title));
 
-  const allExtensions: AnyExtensionData[] = useMemo(
-    () => [...availableExtensionsData, ...installedExtensionsData],
-    [availableExtensionsData, installedExtensionsData],
-  );
-
-  const categorizedExtensions: Record<string, AnyExtensionData[]> =
-    allExtensions.reduce((acc, extension) => {
-      if (!acc[extension.category]) {
-        acc[extension.category] = [];
-      }
-
-      acc[extension.category].push(extension);
-
-      return acc;
-    }, {});
+  const categorizedExtensions = [
+    {
+      category: formatText(MSG.decisionMethods),
+      extensions: getExtensionData(
+        decisionMethodSupportedExtensionsConfig,
+        availableExtensionsData,
+        installedExtensionsData,
+      ).sort((a, b) => formatText(a.name).localeCompare(formatText(b.name))),
```

Comment:
Could we please extract the sorting function into an utility function
```
const sortByName = (extensionA: AnyExtensionData, extensionB: AnyExtensionData) => 
  formatText(extensionA.name).localeCompare(formatText(extensionB.name)))
```
and then use it here as `.sort(sortByName)`